src/partials/id_verification_device.hbs

```diff
@@ -0,0 +1,33 @@
+<div class="card">
+  <div class="org-logo">
+    <img class="org-logo__image" src="{{logo}}" alt="Company Logo"/>
+  </div>
+  <form class="form form--spacing-lg" id="AuthnWidgetForm" style="margin-left:27px;margin-right:27px;">
```

Comment:
Can we add classes for all in line styles and make classes for them in main.scss? We try to steer customizations towards that file so they have a central location

Reply:
Replaced all inline styles in id_verification_*.hbs with classes in main.scss;
The font families defined by default classes in the widget are a bit different than PF template, so some margins/paddings are adjusted to keep the same words per line as in PF template.